js/plot/shared/Plot.svelte

```diff
@@ -1,235 +1,39 @@
 <script lang="ts">
 	//@ts-nocheck
-	import Plotly from "plotly.js-dist-min";
+	import PlotlyPlot from "./plot_types/PlotlyPlot.svelte";
+	import BokehPlot from "./plot_types/BokehPlot.svelte";
+	import AltairPlot from "./plot_types/AltairPlot.svelte";
+	import MatplotlibPlot from "./plot_types/MatplotlibPlot.svelte";
```

Comment:
Since we have split these into separate components, lets lazy load them with a dynamic `import` instead of a static import. That will improve the loading time of apps using `Plot`. No users use all three types of plot at the same time.